internal/test/helpers.go

```diff
@@ -157,6 +158,14 @@ func Ignore(what string) cmp.Option {
 	return cmp.FilterPath(func(p cmp.Path) bool { return p.String() == what }, cmp.Ignore())
 }
 
+func CompareImageTypes() cmp.Option {
```

Comment:
What is the idea behind this function? Although the code seems reasonable, it's not obvious why only these 3 properties were selected for comparison as opposed to e.g. MIMEType or ModulePlatformID. Maybe add a comment explaining it?

Reply:
The idea is that given a triple distro, arch, type the object you get by resolving them should be considered the same, and indeed all the other functions return the same results. I'll add a comment.